lib/storage/s3.js

```diff
@@ -118,9 +119,9 @@ module.exports = function() {
       }
     },
 
-    copyOut: function(path, localPath, options, callback) {
+    streamOut: function(path, options) {
+      const result = new PassThrough();
```

Comment:
Using a PassThrough stream allows us to return a valid stream before we actually have input for it.